R/write.R

```diff
@@ -95,7 +93,7 @@ format_delim <- function(x, delim, na = "NA", append = FALSE, col_names = !appen
   stopifnot(is.data.frame(x))
 
   x <- lapply(x, output_column)
-  stream_delim(x, "", delim, col_names = col_names, append = append, na = na)
+  stream_delim(x, NULL, delim, col_names = col_names, append = append, na = na)
```

Comment:
Unfortunately not, `stdout()` is a text-mode connection and the connections API only works with binary mode connections, you get `'write' not enabled for this connection` errors if you try to use `stdout()` here.